src/app/pages/home.js

```diff
@@ -7,11 +8,21 @@ import WishListContainer from '../components/WishList/WishListContainer';
 
 const HomePage = () => (
   <>
-    <Promotions />
-    <NewArrivalsContainer />
-    <AdvertisingArea />
-    <JoinUs />
-    <WishListContainer />
+    <Flags authorizedFlags={['showPromotions']}>
+      <Promotions />
+    </Flags>
+    <Flags authorizedFlags={['showNewArrivals']}>
+      <NewArrivalsContainer />
+    </Flags>
+    <Flags authorizedFlags={['showAdvertisingArea']}>
+      <AdvertisingArea />
+    </Flags>
+    <Flags authorizedFlags={['showJoinUs']}>
+      <JoinUs />
+    </Flags>
+    <Flags authorizedFlags={['showWishlist']}>
```

Comment:
also pls move `strings` like  `showWishlist` to appConfig object

and use here in such way

`<Flags authorizedFlags={[appConfig.killswitch.wishlist]}>`

Reply:
go to the product item
find the button `add to wish list`
wrap it in Flags tag as well